tests/vite-app/vite.config.mjs

```diff
@@ -15,11 +16,13 @@ export default defineConfig({
   root,
   // esbuild in vite does not support decorators
   esbuild: false,
+  cacheDir: resolve("node_modules", ".vite"),
```

Comment:
do we actually need this change? or is this just a "cosmetic" thing?

Reply:
Yes, it's required, since ut defaults to . embroider directory, which does not exist initially

Reply:
Yea that's a good point, I hadn't considered that 🙃 we can leave this one in then 👍